src/scss/components/_summary.scss

```diff
@@ -35,6 +35,11 @@
     border-block-end-color: var(--of-summary-header-border-block-end-color, var(--of-color-border));
   }
 
+  // accessible description
+  @include bem.element('link-description') {
+    display: none;
```

Comment:
I thought that `display: None` is not considered as accessible. I tried this though with a screenreader as well.

Reply:
discussed in office - confirmed to work and I did find some article explaining this is okay because of the explicit `aria-describedby`